src/Task.ts

```diff
@@ -109,6 +109,10 @@ export class Task {
     // description: '#dog #car http://www/ddd#ere #house'
     // matches: #dog, #car, #house
     public static readonly hashTags = /(^|\s)#[^ !@#$%^&*(),.?":{}|<>]*/g;
+    public static readonly hashTagsFromEnd = new RegExp(
+        this.hashTags.source + '$',
+        'g',
```

Comment:
What does the `'g'` do here? The regex ends with `$` so can only ever match one instance, the last one. It's harmless, obviously, but I was just wondering about the intention.

Reply:
It's redundant, removing to prevent confusion.